README.md

```diff
@@ -207,6 +207,15 @@ NOTE: Enable Ambassador/Nginx in development mode, if Appstore is running on loc
 ```
 helm install release-name $HELXPLATFORM_HOME/devops/helx --values basic-values.yaml --namespace your-k8s-namespace
 ```
+
+### TLS/SSL
+
+The helm charts will not create the TLS secrets for nginx (`nginxTLSSecret`), meaning the nginx pod will crashloop until that secret is created. You will either need to create a self-signed certificate, or have an administrator create one for you. To create the secret from an existing certificate:
+
+```
+kubectl create secret tls -n <your-namespace> <your-namespace>.blackbalsam-cluster.edc.renci.org-tls-secret --cert=tls.crt --key=tls.key
+```
+
```

Comment:
This is good info, maybe it should be in the nginx chart?  This also doesn't sound like the way the nginx chart works.  If the TLS secret name is not defined then the TLS lines should not be included in the nginx config file.  Also, the README.md is auto-generated by [helm-docs]( https://github.com/norwoodj/helm-docs), so should add this info in README.gotmpl (see pod-reaper chart for an example).

Reply:
Ah, will remove the part about the crashlooping. I see https://help.renci.org/Ticket/Display.html?id=4169 was actually caused by Bo setting nginxTLSSecret to a secret name that didn't exist, rather than leaving it blank.